### PR TITLE
feat: sqlalchemy tests

### PR DESCRIPTION
**Pull request by jarulraj**
- [X] Verify tables and services
- [X] Only load `YOLO` is particular test cases (to avoid slowing down all test cases)

### CODE REVIEW

File: test/catalog/test_sqlalchemy.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                self.assertTrue(len(column_infos) < 10, f"{table} has too many columns")
+                for column_info in column_infos:
+                    print(column_info["name"], end=" | ")
+
+            service_subclasses = get_all_subclasses(BaseService)
+            for service_subclass in service_subclasses:
+                service = service_subclass()
+                table_tuples = service.get_all_entries()
+                self.assertTrue(
+                    len(table_tuples) < 100
```
Comment (jiashenC) [inline test/catalog/test_sqlalchemy.py:47]: Just curious, is it generally bad to have a large catalog table?

**Review by jiashenC — APPROVED** [review]

File: test/catalog/test_sqlalchemy.py (hunk 1 above)
Comment (jarulraj) [reply test/catalog/test_sqlalchemy.py:47]: Not really, I just added an assert for checking the contents of the table for now.

File: eva/udfs/udf_bootstrap_queries.py [hunk 2]
```diff
@@ -179,10 +179,10 @@ def init_builtin_udfs(mode="debug"):
         ArrayCount_udf_query,
         Timestamp_udf_query,
         Crop_udf_query,
-        YoloV5_udf_query,
         Open_udf_query,
         Similarity_udf_query
         # Disabled because required packages (eg., easy_ocr might not be preinstalled)
+        # YoloV5_udf_query,
```
[the remark this answers is not in the document]
Comment (jarulraj) [reply eva/udfs/udf_bootstrap_queries.py]: Done. We now have three modes: debug/release: all UDFS, minimal: No YOLO

File: test/benchmark_tests/conftest.py [hunk 3]
```diff
@@ -25,5 +25,8 @@ def setup_pytorch_tests():
     CatalogManager().reset()
     execute_query_fetch_all("LOAD VIDEO 'data/ua_detrac/ua_detrac.mp4' INTO MyVideo;")
     execute_query_fetch_all("LOAD VIDEO 'data/mnist/mnist.mp4' INTO MNIST;")
-    load_inbuilt_udfs()
+    load_udfs_for_testing()
+    from eva.udfs.udf_bootstrap_queries import YoloV5_udf_query
```
Comment (gaurav274) [inline test/benchmark_tests/conftest.py:29]: Any reason for having this separately?

Comment (jarulraj) [reply test/benchmark_tests/conftest.py:29]: In case we want a more complex logic for picking UDFs later during `testing`?

**Pull request merged**